### PR TITLE
Update BYOC docs on how to create tenant cluster on a different tenant

### PR DESCRIPTION
**Pull request by fiunchinho**
In case we change the way BYOC works on Azure, the docs need to be updated.

The new mechanism works differently.
A new multi-tenant service principal is created for every installation, and the same service principal is used to create both the control plane and tenant clusters. In order to do that, the service principal needs to be given permissions in the subscription where the tenant cluster will run (this is the same as before). If the subscription is linked to a different tenant AD than the control plane subscription, the service principal needs to be invited to the tenant AD.

### CODE REVIEW

File: src/content/guides/prepare-azure-subscription-for-tenant-clusters/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Prepare an Azure subscription to run Giant Swarm tenant clusters"
-description: "This guide will walk you through all necessary steps to set up an Azure subscription with approriate Role definition and Service Principal for operating Giant Swarm tenant clusters."
+description: "This guide will walk you through all necessary steps to set up an Azure subscription for operating Giant Swarm tenant clusters."
 date: "2018-09-12"
```
Comment (axbarsan) [inline src/content/guides/prepare-azure-subscription-for-tenant-clusters/index.md:4]: Don't forget to update the date 👍

Comment (fiunchinho) [reply src/content/guides/prepare-azure-subscription-for-tenant-clusters/index.md]: Done

File: src/content/guides/prepare-azure-subscription-for-tenant-clusters/index.md [hunk 2]
```diff
@@ -1,30 +1,33 @@
...
+date: "2020-05-14"
 type: page
 weight: 100
 tags: ["tutorial"]
 ---
 
 # Prepare an Azure subscription to run Giant Swarm tenant clusters
 
-In a Giant Swarm installation the tenant clusters (the clusters running your Kubernetes workloads) can run in a separate Azure subscription from the control plane.
+In a Giant Swarm installation the tenant clusters (the clusters running your Kubernetes workloads) can run in a separate
```
Comment (axbarsan) [inline src/content/guides/prepare-azure-subscription-for-tenant-clusters/index.md:12]:
```suggestion
In a Giant Swarm installation, the tenant clusters (the clusters running your Kubernetes workloads) can run in a separate
```

**Comment by oponder** [comment]
Can this be merged? Don't worry about the linkchecker failure in CI

**Comment by fiunchinho** [comment]
Closing until we figure out how to do it.

File: src/content/guides/prepare-azure-subscription-for-tenant-clusters/index.md [hunk 3]
```diff
@@ -1,98 +1,102 @@
...
+In that case, please hand over your Azure subscription ID and Tenant ID to your partner contact.
 
-After the organization's credentials are set, you can create clusters owned by that
-organization. These clusters' resources will be created in your Azure subscription.
+After the organization is configured, you can create clusters owned by that organization.
+These clusters' resources will be created in your Azure subscription.
 
 ## Further reading
 
 - [Basics and Concepts: Bring Your Own Cloud](/basics/byoc/)
```
Comment (marians) [inline src/content/guides/prepare-azure-subscription-for-tenant-clusters/index.md:100]:
```suggestion
- [Basics and Concepts: Bring Your Own Cloud](/basics/multi-account/)
```

**Comment by marians** [comment]
FYI: "BYOC" as a term is no longer used. We call it "Multi account" by now.

**Comment by marians** [comment]
Want to delete that branch `byoc-azure`?

**Pull request closed without merging**